README.md

```diff
@@ -44,7 +44,7 @@ pip install feast
 ### 2. Create a feature repository
 ```commandline
 feast init my_feature_repo
-cd my_feature_repo/feature_repo
+cd my_feature_repo
```

Comment:
This was generated by pre commit hook?

Reply:
ah would you mind reverting this and fixing this in https://github.com/feast-dev/feast/blob/master/infra/templates/README.md.jinja2#L45?